_tutorial_base/800-ci.md

```diff
@@ -18,13 +18,13 @@ So we only have to fire up a shell and run the build!
 
 You do not even need to have gradle installed. There is a script included, `gradlew`, that will download the correct gradle version. You can also install gradle (>= 1.12) and use it. It should go without saying, but you should also have java 8 installed on the command line.
 
-	$ cd /Ws/com.acme.prime
 	$ java -version
 	java version "1.8.0"
+	$ cd /Users/aqute/git/com.acme.prime
```

Comment:
Amended my last comment on my local repo but having difficulty doing a
git push --force origin master 
to gitHub.  Any suggests or can I just add another commit noting that previous commits are signed off?